js/components/fetch-token.mjs

```diff
@@ -0,0 +1,144 @@
+/** 
+ * Get accessToken from local storage
+ * Create functions to API with header included token
+ */
+const givenToken = localStorage.getItem("accessToken");
+
+// GET header
+const getData = {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${givenToken}`,
+    },
+  };
+
+
+
+/**
+ * Get all Feed posts
+ * @date 2023-10-12
+ * @param {string} url This is an API link to get posts
+ * @returns {Array} This is an array of post objects. Later using these as param in showPosts function
+ */
+async function getPosts(url) {
+try {
+    getData;
+    const response = await fetch(url, getData);
+    const posts = await response.json();
+    return posts;
+} catch (error) {
+    console.log(error);
```

Comment:
I recommend removing this try/catch block. It's best to handle errors close to the user, for example in a form handler or render method.

Reply:
I solved this, thanks!